build_cmake/toolchains/aarch64/linux/plat_clang.cmake

```diff
@@ -0,0 +1,15 @@
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#  All rights reserved.
+#
+# See the file LICENSE for redistribution information.
+#
+
+if(CMAKE_CROSSCOMPILING)
+    set(TRIPLE_TARGET "aarch64-unknown-linux-gnu")
```

Comment:
I guess TRIPLE_TARGET indicates the target triplet <machine>-<vendor>-<os>. Is `linux-gnu` treated as the os part (feel like a quartet to me in the 1st glance)?

Reply:
True, its a bit confusing. I was following clang terminology here, where they the technical term for the string is a 'triple' (https://clang.llvm.org/docs/CrossCompilation.html#general-cross-compilation-options-in-clang). 
A triple consists of `<arch><sub>-<vendor>-<sys>-<abi>` (so yep, sys-abi are one element).